runtime/CMakeLists.txt

```diff
@@ -50,6 +50,21 @@ if(ENABLE_CODE_COVERAGE)
     target_link_libraries(catalyst_qir_runtime INTERFACE gcov)
 endif()
 
+if(ENABLE_SANITIZER)
+    message(STATUS "ENABLE_SANITIZER is ON.")
+    if(CMAKE_C_COMPILER_ID STREQUAL "GNU")
+        add_compile_options(-fsanitize=address)
+        add_link_options(-fsanitize=address -static-libasan)
+    else()
```

Comment:
Should this be an "else if" if it is specific to clang?

Reply:
Hmm... yes, better to use `else if` and check if the compiler is `clang` instead of assuming it :)  I'll fix this before merging. Thanks!